packages/radio-group/src/styles.ts

```diff
@@ -0,0 +1,252 @@
+import { css } from '@leafygreen-ui/emotion';
+import { createUniqueClassName, Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import {
+  fontFamilies,
+  focusRing,
+  hoverRing,
+  typeScales,
+} from '@leafygreen-ui/tokens';
+import { Size } from './types';
+
+export const inputDisplay = createUniqueClassName('radio-group');
+export const inputDisplayWrapper = createUniqueClassName('radio-group');
+export const inputDataProp = createUniqueClassName('radio-group');
+
+export const containerMargin = css`
+  & + & {
+    margin-top: 8px;
+  }
+`;
+
+export const offsets: Record<Size, string> = {
+  [Size.XSmall]: css`
+    margin-top: -3px;
+    margin-left: 4px;
+  `,
+  [Size.Small]: css`
+    margin-top: -3px;
+    margin-left: 8px;
+  `,
+  [Size.Default]: css`
+    margin-top: 0;
+    margin-left: 8px;
+  `,
+};
```

Comment:
Why are we using negative margin instead of flexbox for vertical alignment? 
Also, if we stick with this, I think `offsets` should have a more descriptive name

Reply:
Depending on the `Size` the radio height changes and the text font size stays the same. If there was only one line of text we could use flexbox to center it but if there are multiple lines the radio would no longer center align with the first line of text.

Reply:
Ah, makes sense 👍